genesyscloud/outbound_callanalysisresponseset/resource_genesyscloud_outbound_callanalysisresponseset_utils.go

```diff
@@ -94,14 +104,16 @@ func flattenSdkReaction(sdkReaction platformclientv2.Reaction) *schema.Set {
 	return reactionSet
 }
 
-func GenerateOutboundCallAnalysisResponseSetResource(resourceLabel string, name string, beepDetectionEnabled string, responsesBlock string) string {
+func GenerateOutboundCallAnalysisResponseSetResource(resourceLabel string, name string, beepDetectionEnabled string, amdSpeechDistinguishEnabled string, liveSpeakerDetectionMode string, responsesBlock string) string {
 	return fmt.Sprintf(`
 resource "genesyscloud_outbound_callanalysisresponseset" "%s" {
 	name                   = "%s"
 	beep_detection_enabled = %s
+	amd_speech_distinguish_enabled = %s
+	live_speaker_detection_mode = "%s"
```

Comment:
Could you change this to `live_speaker_detection_mode = %s` and where ever you call `GenerateOutboundCallAnalysisResponseSetResource` , provide `"null"` or `strconv.Quote("the value")` ? 

In the first test step in TestAccResourceOutboundCallAnalysisResponseSet , it's better to keep both these new fields `null` , verify they're `null`, and then update them in the next step. This way we know existing configs won't be broken when this change is released